megalinter/descriptors/logstash.megalinter-descriptor.yml

```diff
@@ -0,0 +1,32 @@
+descriptor_id: LOGSTASH
+descriptor_type: tooling_format
+file_extensions:
+  - ".conf"
```

Comment:
I'm concerned here that having this descriptor enabled in a vanilla config with the full megalinter might run on unexpected files. `.conf` isn't an extension unique to logstash. 

Will the linter, if ran on .conf files that aren't logstash config files, but any other config file, will throw false positives for everyone else?

Reply:
Hmm good question, it will probably throw indeed... It's a fairly specific linter so indeed should probably be opt-in only, but not sure how that can be handled?

Reply:
By convention, Logstash conf is often under a `**/pipeline` directory, so maybe we can filter on this, but not sure how I can specify this?

Reply:
> By convention, Logstash conf is often under a `**/pipeline` directory, so maybe we can filter on this, but not sure how I can specify this?

You can try with syntax like used in `actionlint` linter (see https://github.com/oxsecurity/megalinter/blob/main/megalinter/descriptors/action.megalinter-descriptor.yml#L5)

Reply:
OK, I've added this! I've set it to `logstash` by default, which I guess may not generally be appropriate, but at least should avoid conflicts with other formats by default, and can be overridden to suit specific cases.